.github/workflows/ci.yml

```diff
@@ -59,6 +59,8 @@ jobs:
         run: |
           if [[ ${{ matrix.engine }} == "trino" || ${{ matrix.engine }} == "starburst" ]]; then
             make dbt-${{ matrix.engine }}-tests
+          elif [[ ${{ env.DBT_TESTS_STARBURST_GALAXY_HOST == '' }} || ${{ env.DBT_TESTS_STARBURST_GALAXY_USER == '' }} || ${{ env.DBT_TESTS_STARBURST_GALAXY_PASSWORD == '' }} ]]; then
```

Comment:
Does this mean that we have an empty check for galaxy on PRs coming from forks?
IMO the check shouldn't appear at all in such cases.

Reply:
Can you try the this way: https://github.com/orgs/community/discussions/26253#discussioncomment-3250989